compute/cker/include/cker/operation/Conv.h

```diff
@@ -121,9 +117,10 @@ class Conv
                        params.stride_height);
     }
 
-    uint8_t *im2col_raw_data = _im2col_data.data();
+    int im2col_size = _need_im2col ? _im2col_shape.FlatSize() : 1;
+    uint8_t im2col_data[im2col_size];
```

Comment:
@hseok-oh `im2col_data` will be allocated from stack at runtime. Allocation may fail if `im2col_size` is big.

Reply:
OK. I'll use heap for this.